flink-core/src/test/java/org/apache/flink/util/ReferenceTest.java

```diff
@@ -18,39 +18,39 @@
 
 package org.apache.flink.util;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the {@link Reference}. */
 @ExtendWith({TestLoggerExtension.class})
```

Comment:
TestLoggerExtension has been enabled at module level, so it can be removed here.